Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.10.11
```

Comment:
Hi, I'd suggest having a look at https://github.com/GoogleContainerTools/distroless too

It provides different base images, python3 included, that are very small and already has a user inside them. It could be very effective to slim down the image size as much as possible!

Reply:
Hello I already tried some light/distroless image, but the requirements.txt get a lot of dependency like 8Go and need GCC compiler, eventually cuda and other stuff.

You will win like 200M (2%) of the total image size and will probably need to add gcc/python-dev/cuda manually

If you have a working dockerfile using a distroless/light don't hesitate to contribute

Reply:
Hello I already tried some light/distroless image, but the requirements.txt get a lot of dependency like 8Go and need GCC compiler, eventually cuda and other stuff.

You will win like 200M (2%) of the total image size and will probably need to add gcc/python-dev/cuda manually

If you have a working dockerfile using a distroless/light don't hesitate to contribute

Reply:
Yeah, you're right, I didn't take in consideration the CUDA dependencies that are very heavy. It'd make a little sense, size wise. Maybe security wise could have some sense though, because there is no default shell. But I guess it depends on which environment you are going to use it (and since it is a cli, I think it doesn't really make sense).